Integrations/AWS-EC2/AWS-EC2.py

```diff
@@ -0,0 +1,2679 @@
+
+
+def create_image_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+    obj = vars(client._client_config)
+    kwargs = {
+        'Name': args.get('name'),
+        'InstanceId': args.get('instanceId')
+    }
+
+    if args.get('description') is not None:
+        kwargs.update({'Description': args.get('description')})
+    if args.get('noReboot') is not None:
+        kwargs.update({'NoReboot': True if args.get('noReboot') == 'True' else False})
+
+    response = client.create_image(**kwargs)
+
+    data = ({
+        'ImageId': response['ImageId'],
+        'Name': args.get('name'),
+        'InstanceId': args.get('instanceId'),
+        'Region': obj['_user_provided_options']['region_name'],
+    })
+
+    ec = {'AWS.EC2.Images': data}
+    human_readable = tableToMarkdown('AWS EC2 Images', data)
+    return_outputs(human_readable, ec)
+
+
+def deregister_image_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    response = client.deregister_image(ImageId=args.get('imageId'))
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+        demisto.results("The AMI with ID: {0} was deregistered".format(args.get('imageId')))
+
+
+def modify_volume_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    obj = vars(client._client_config)
+    kwargs = {'VolumeId': args.get('volumeId')}
+
+    if args.get('size') is not None:
+        kwargs.update({'Size': int(args.get('size'))})
+    if args.get('volumeType') is not None:
+        kwargs.update({'VolumeType': args.get('volumeType')})
+    if args.get('iops') is not None:
+        kwargs.update({'Iops': int(args.get('iops'))})
+
+    response = client.modify_volume(**kwargs)
+    volumeModification = response['VolumeModification']
+
+    data = ({
+        'VolumeId': volumeModification['VolumeId'],
+        'ModificationState': volumeModification['ModificationState'],
+        'TargetSize': volumeModification['TargetSize'],
+        'TargetIops': volumeModification['TargetIops'],
+        'TargetVolumeType': volumeModification['TargetVolumeType'],
+        'OriginalSize': volumeModification['OriginalSize'],
+        'OriginalIops': volumeModification['OriginalIops'],
+        'OriginalVolumeType': volumeModification['OriginalVolumeType'],
+        'StartTime': datetime.strftime(volumeModification['StartTime'], '%Y-%m-%dT%H:%M:%SZ'),
```

Comment:
this can throw value error. better calculate it before the append and in a try block.